tests/js/browser-binder-tests.js

```diff
@@ -0,0 +1,639 @@
+// Test "binder" components using `gpii-test-browser`.
+"use strict";
+var fluid = require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+
+require("gpii-test-browser");
+gpii.tests.browser.loadTestingSupport();
+
+var path    = require("path");
+var url     = require("url");
+var fileUrl = url.resolve("file://", path.resolve(__dirname, "../static/tests-binder.html"));
+
+fluid.registerNamespace("gpii.binder.tests");
+
+// TODO:  If we find ourselves making model changes often, move this to `gpii-test-browser` and polish it up for wider use.
+gpii.binder.tests.applyChange = function (componentPath, modelPath, newValue) {
+    var component = fluid.getGlobalValue(componentPath);
+    component.applier.change(modelPath, newValue);
+};
+
+
+fluid.defaults("gpii.binder.tests.caseHolder", {
```

Comment:
Do we test anything within this fixture that couldn't be simulated from within the browser? Given there is no server bound here, it seems that it would improve the portability and speed of these tests if they were just run as standard browser tests.